### DISCUSSION TITLE
pt_BR Translation

### DISCUSSION THREAD

**Discussion by thiagojramos** [LykosAI/StabilityMatrix#277] [General]
Hello, how can I help to translate into Brazilian Portuguese? My email: [thiagojramos@outlook.com](mailto:thiagojramos@outlook.com)

**Comment by thiagojramos** [comment]
🥱

**Comment by mohnjiles** [comment]
Thanks for the interest, and sorry for the delay! We've been having issues adding new collaborators to our translation project. We will send an invite as soon as it is resolved.
